dev/CheckBox/CheckBox_themeresources.xaml

```diff
@@ -302,8 +392,7 @@
                         Background="{TemplateBinding Background}"
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
-                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                        contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
```

Comment:
Is there a reason the Contract7NotPresent scenario is removed?

Reply:
It's a bug, we don't need corner radius in before contract7 scenarios. cleaned this up in other controls I worked on as well.